app_legacy/Helpers/render/avatar.php

```diff
@@ -20,7 +20,7 @@ function avatar(
     }
 
     if ($iconUrl) {
-        $iconLabel = "<img loading='lazy' width='$iconSize' height='$iconSize' src='$iconUrl' alt='$escapedName' class='$iconClass'>";
+        $iconLabel = "<img loading='lazy' decoding='async' fetchpriority='low' width='$iconSize' height='$iconSize' src='$iconUrl' alt='$escapedName' class='$iconClass'>";
```

Comment:
Gotta make sure this snippet of code doesn't have unintended side-effects, given that it's used to generate most of the game/cheevo images on the site.

Reply:
I have reverted this change

Reply:
Would it be useful to have an extra parameter so those attributes are applied just in this particular case and not generally?